--- dbus-modbus-client.py_org
+++ dbus-modbus-client.py
@@ -27,5 +27,6 @@
 import abb                                                                                                                                              
 import carlo_gavazzi
+import Eastron_SDM72D
 import comap                                                                                                                                            
 import cre                                                                                                                                              
 import deif
